package.json

```diff
@@ -7,7 +7,11 @@
     "test": "cypress run --headless --browser chrome",
     "test:open": "npm run lint && cypress open",
     "lint": "eslint ./cypress/e2e/**/*.ts",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports",
+    "runtest": "npm run test && npm run combine:reports && npm run generate:reports",
```

Comment:
this should be the `posttest` script, with just this content `npm run combine:reports && npm run generate:reports`